views/user_info.erb

```diff
@@ -1,8 +1,8 @@
 <div class="p2 bg-green white clearfix">
   <% if userinfo %>
-    <form action="/auth/saml/logout" method="POST" class="right sm-show">
-      <button type="submit" class="btn p0 h6 white">Log out</button>
-    </form>
+    <div class="right sm-show">
+      <a href="/" class="btn p0 h6 white">Log out</button>
```

Comment:
This is how it works. 😬  The app currently doesn't maintain any sort of session or cookie or anything. Being "logged in" is having an authorization code in the URL, so linking back to the home page without a param shows a logged out page #hacks